support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -75,8 +89,8 @@ export type RegularPaymentRequest = {|
   deliveryAddress: Option<RegularPaymentRequestAddress>,
   email: string,
   titleGiftRecipient?: Option<Title>,
-  firstNameGiftRecipient?: string,
-  lastNameGiftRecipient?: string,
+  firstNameGiftRecipient?: Option<string>,
+  lastNameGiftRecipient?: Option<string>,
```

Comment:
what does it mean if it's both optional `?` and an option `Option`?

Reply:
if this is a "payment request", surely the gift status is not relevant? payment would just be for billing info/frequency/amount and payment card number?

Reply:
It can be missing entirely and if it is present it can be null

Reply:
sorry I wasn't clear, I meant what does it mean in real life!  I know what it means in terms of the code.
I guess missing means it's not a gift, and null means it wasn't entered by the gifter?